scripts/test_e2e.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright 2019 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+service mysql start
+
+
+go generate ./...
+go get -v -t ./...
+go install ./...
+
+DATASOURCE="mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0"
+sqlflowserver --datasource=${DATASOURCE} &
```

Comment:
Need to kill this `sqlflowserver` process when the test is finished, because mutiple test scripts need to run.

Reply:
Looks like `test_e2e.sh` is ran in an isolated container. I am wondering: after the execution of `ipython sql/python/test_magic.ipy`, will Docker stop the container by killing all the running process?

Reply:
Looks like `test_e2e.sh` is ran in an isolated container. I am wondering: after the execution of `ipython sql/python/test_magic.ipy`, will Docker stop the container by killing all the running process?

Reply:
Yes, The Travis would run these test scripts in the separate Docker containers, maybe we can execute these test in one Docker container in the future.